miio/airpurifier.py

```diff
@@ -209,6 +214,27 @@ def learn_mode(self) -> bool:
         """Return True if Learn Mode is enabled."""
         return self.data["act_sleep"] == "single"
 
+    @property
+    def sleep_time(self) -> Optional[int]:
+        return self.data["sleep_time"]
+
+    @property
+    def sleep_mode_learn_count(self) -> Optional[int]:
+        return self.data["sleep_data_num"]
+
+    @property
+    def extra_features(self) -> Optional[int]:
+        return self.data["app_extra"]
+
+    @property
+    def turbo_mode_supported(self) -> Optional[bool]:
+        return self.data["app_extra"] == 1
```

Comment:
This will return a bool for any value of `app_extra`, so no need to mark it Optional here.